### PR TITLE
Fix for Users and Content Tests

### PR DESCRIPTION
**Pull request by puneet0191**

### CODE REVIEW

File: src/acceptance/administrator/components/com_users/UserCest.php [hunk 1]
```diff
@@ -16,6 +16,16 @@
  */
 class UserCest
 {
+
+	public function __construct()
+	{
+		$this->faker = Faker\Factory::create();
+		$this->username = $this->faker->userName();
```
Comment (yvesh) [inline src/acceptance/administrator/components/com_users/UserCest.php]: Remove Faker ;)

File: src/acceptance/administrator/components/com_users/UserCest.php [hunk 2]
```diff
@@ -69,14 +79,14 @@ public function editUser(\AcceptanceTester $I)
...
+		$I->click($this->name);
 
 		$I->waitForElement(Administrator\UserManagerPage::$accountDetailsTab);
 		$I->checkForPhpNoticesOrWarnings();
 
-		$this->fillUserForm($I, \Shared\UserCredentials::$name, \Shared\UserCredentials::$username, \Shared\UserCredentials::$password, \Shared\UserCredentials::$email);
+		$this->fillUserForm($I, $this->name, $this->username, $this->password, $this->email);
 
-		$I->click(Administrator\UserManagerPage::$saveButton);
+		$I->clickToolbarButton("Save");
```
Comment (yvesh) [inline src/acceptance/administrator/components/com_users/UserCest.php:88]: Why not anymore from the page object?

File: src/acceptance/administrator/components/com_users/UserCest.php (hunk 1 above)
Comment (puneet0191) [reply src/acceptance/administrator/components/com_users/UserCest.php]: :(

File: src/acceptance/administrator/components/com_users/UserCest.php (hunk 2 above)
Comment (puneet0191) [reply src/acceptance/administrator/components/com_users/UserCest.php:88]: Joomla Browser

**Pull request merged**